shim/shim.c

```diff
@@ -94,30 +94,6 @@ signal_handler(int signal_no)
 	errno = savedErrno;
 }
 
-/*!
- * Assign signal handler for all the signals that should be
- * forwarded by the shim to the proxy.
- *
- * \param sa Signal handler
- * \return true on success, false otherwise
- */
-bool
-assign_all_signals(struct sigaction *sa)
```

Comment:
cc-oci-runtime will send signals to hyper directly because '--all' option was added to kill command hence cc-shim doesn't know when kill just one process or when kill all processes